aip/client-libraries/4221.md

```diff
@@ -112,9 +112,10 @@ explicitly named which get no `RetryPolicy`.
   }]
 }
 ```
+<!-- prettier-ignore-end -->
```

Comment:
I'm assuming *all* JSON blocks must be ignored. How will folks know to use these tags around JSON blocks (maybe even code blocks)? Catch it in review? Does it matter?

Reply:
I do not think it will matter most of the time, honestly.